packages/clerk-js/src/ui/elements/ArrowBlockButton.tsx

```diff
@@ -73,14 +73,21 @@ export const ArrowBlockButton = (props: ArrowBlockButtonProps) => {
         <Flex
           as='span'
           center
-          sx={theme => ({ flex: `0 0 ${theme.space.$4}` })}
+          sx={[
+            theme => ({
+              flex: `0 0 ${theme.space.$4}`,
+            }),
+            theme =>
+              isLoading && {
+                padding: theme.space.$0x5,
+              },
+          ]}
         >
           {isLoading ? (
             <Spinner
               elementDescriptor={spinnerElementDescriptor}
               elementId={spinnerElementId}
-              sx={{ position: 'absolute' }}
-              size={'sm'}
+              size={'md'}
```

Comment:
🙃 
Can we explore slightly changing the `Spinner` component here to avoid the `isLoading` check? It feels like the icon and the Spinner should have the exact same dimensions so one could be rendered in place of the other one without layout swift

Reply:
@nikosdouvlis Just changed it and added padding to the `Spinner` component to avoid the `isLoading` check.